README.md

```diff
@@ -1 +1,80 @@
-# HTML-Forms
\ No newline at end of file
+# Embedded HTML/CSS Project
+
+![screenshot](images/HTML-form-2nd-project.png)
+
+### **Description:** 
+
+Project based on Mint's sign-up page, it shows a Sign-in section on the top-right corner, a section with a Form for inputting data and related information, and a footer section.
+ 
+ **Header (Sign-in section):**
+ 
+ 	- A link to another URL (if user forgot User ID or password).
+ 	- A button for Signing in.
+ 	
+ **Form section:**
+ 
+ 	- Embedded images for the logo of the site and for three different URLs for further information.
+ 	- An HTML form for inputting data (Email, phone, and password).
+ 	- A button for creating the account.
+ 	- Sections with information including URLs to redirect to those websites.
+ 	
+ **Footer section:**
+ 
+ 	- Three URLs redirecting to Legal, Privacy, and Security sites with further information.
+ 	- Text related to legal information about the website.
+ 	
+	
+ ### **Tools used:**
+ 
+ HTML5, and CSS3.
+ 
+ 
+ 
+ ### **Live Demo link:** 
+ 
+ https://ultramax.github.io/HTML-Forms/
+
+ 
```

Comment:
Please create a link in your README file, you can use the same syntax as with your screenshot.
You can use the brackets `[Live demo link]()`